tests/integration/support/test_support.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+
+from tests.integration.helpers import assert_headers_in_lines, exec_test_command
+
+BASE_CMD = ["linode-cli", "tickets"]
+
+
+# this will create a support ticket on your account
```

Comment:
Can we have a cleanup/teardown for the tickets that are created?

Reply:
tickets created won't be able to delete form command, while clean-up we need to mark them close 
 we have  command - "linode-cli tickets close ticket_id" it gives request failed error